software/control/gui_hcs.py

```diff
@@ -499,6 +524,8 @@ def setupCameraTabWidget(self):
             self.cameraTabWidget.addTab(self.spinningDiskConfocalWidget, "Confocal")
         if USE_ZABER_EMISSION_FILTER_WHEEL or USE_OPTOSPIN_EMISSION_FILTER_WHEEL:
             self.cameraTabWidget.addTab(self.filterControllerWidget, "Emission Filter")
+        if USE_SQUID_FILTERWHEEL == True:
```

Comment:
Just out of curiosity, why all the different ways of `if USE_SQUID_FILTERWHEEL:`?  So far there are 3 in this chunk of code!

Reply:
@veerwang

Reply:
I would modify it.